com.unity.render-pipelines.universal/Runtime/ForwardLights.cs

```diff
@@ -75,6 +98,201 @@ public ForwardLights()
                 m_AdditionalLightOcclusionProbeChannels = new Vector4[maxLights];
                 m_AdditionalLightsLayerMasks = new float[maxLights];
             }
+
+            if (m_UseClusteredRendering)
+            {
+                m_ZBinBuffer = new ComputeBuffer(UniversalRenderPipeline.maxZBins / 4, UnsafeUtility.SizeOf<float4>(), ComputeBufferType.Constant, ComputeBufferMode.Dynamic);
+                m_TileBuffer = new ComputeBuffer(UniversalRenderPipeline.maxTileVec4s, UnsafeUtility.SizeOf<float4>(), ComputeBufferType.Constant, ComputeBufferMode.Dynamic);
+                m_RequestedTileWidth = tileSize;
+            }
+        }
+
+        internal void ProcessLights(ref RenderingData renderingData)
+        {
+            if (m_UseClusteredRendering)
```

Comment:
I think this is a bit dangerous design of function, as it only executes anything if m_UseClusteredRendering is true. Which could potentially hide issues if it would be called somewhere else as empty and expected it would do smth or not.

Reply:
I'm not sure I follow. What issue could it hide?

Reply:
I think we can do this in follow up pr. This is no real issue, just more like clear code thingy, as you can call function that will not do anything with m_UseClusteredRendering false. My recommendation would be rename this function to ProcessClusteredLights and remove this condition and call it only in clustered light context as it would be a bit clear.

Reply:
Ah okay. I can also make it optional to call, so that if you don't call it, then `Setup` will do it. Then it's just an opportunity for spawning jobs early. Maybe rename it `SpawnJobs`.

Not a huge fan of moving the branch outside though, as calling it when clustering is not enabled would cause leaks.

But yeah, let's handle this in a follow-up PR :)